packages/forge2d/lib/src/dynamics/joints/friction_joint.dart

```diff
@@ -223,4 +223,17 @@ class FrictionJoint extends Joint {
   bool solvePositionConstraints(final SolverData data) {
     return true;
   }
+
+  @override
+  void render(DebugDraw debugDraw) {
+    super.render(debugDraw);
+
+    final xf1 = bodyA.transform;
+    final xf2 = bodyB.transform;
+    final x1 = xf1.p;
+    final x2 = xf2.p;
+    final color = Color3i.zero()..setFromRGBd(0.5, 0.8, 0.8);
```

Comment:
You can create this object outside of the render method so that it doesn't have to be recreated on every tick.